examples/quickstart/README.md

```diff
@@ -0,0 +1,87 @@
+# Example setup
+
+This example shows the bare minimum requirements to deploy applications on wasmCloud.
+
+It relies on the Kubernetes `default` namespace for simplicity.
+
+## Install [NATS](https://github.com/nats-io/nats-server)
+
+```bash
+helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+helm upgrade --install -f nats-values.yaml nats-cluster nats/nats
```

Comment:
I think we should rename the helm NATS deployment from `nats-cluster` to just `nats`, the historical reasons for why we went with nats-cluster no longer apply:
```suggestion
helm upgrade --install -f nats-values.yaml nats nats/nats
```